src/platforms/apple/common/configuration/metric-kit.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title:  Metric Kit
+sidebar_order: 20
+description: "This feature, once enabled, subscribes to MXDiagnosticPayload data of MetricKit and it to Sentry."
+---
+
+<Include name="alpha-note.mdx" />
+
+Once enabled, this feature subscribes to [MXDiagnosticPayload](https://developer.apple.com/documentation/metrickit/mxdiagnosticpayload) data of [MetricKit](https://developer.apple.com/documentation/metrickit), converts it to events and sends it to Sentry. This feature is available on Cocoa 8.0.0 and above. The MetricKit integration subscribes to
+
+* [MXHangDiagnostic](https://developer.apple.com/documentation/metrickit/mxhangdiagnostic)
+* [MXDiskWriteExceptionDiagnostic](https://developer.apple.com/documentation/metrickit/mxdiskwriteexceptiondiagnostic)
+* [MXCPUExceptionDiagnostic](https://developer.apple.com/documentation/metrickit/mxcpuexceptiondiagnostic)
+
+The SDK supports this feature from iOS 15 and later and macOS 12 and later because, on these versions, MetricKit delivers diagnostic reports immediately, which allows the Sentry SDK to apply the current data from the scope.
+
```

Comment:
```suggestion
The SDK supports this feature from iOS 15 and up and macOS 12 and up because in these versions, MetricKit delivers diagnostic reports immediately, which allows the Sentry SDK to apply current data from the scope.

```